src/components/BuyModelInput.js

```diff
@@ -77,36 +77,36 @@ export default ({
           <Input
             style={inputStyle}
             type="number"
-            placeholder="Monthly Patronage"
-            onChange={eventToValue(updatePatronage)}
-            value={patronage}
+            placeholder="Your Initial Sale Price"
+            onChange={eventToValue(setNewPrice)}
+            value={newPrice}
           />
         </Box>
         <Box p={2} mb={2} width={[1, 0.5]}>
           <Tooltip
-            message={`This is the amount of money you will receive if someone purchases ${animalName} from you.`}
+            message="This is the monthly contribution that will go towards conservation of endangered animals. This will be deducted continuously from your deposit"
             placement="top"
           >
             <Text style={rightAlignedText}>
-              {animalName}'s new for sale price:
+              Your monthly contribution:
               <div style={infoTooltipStyle}>
                 <Icon color="green" name="Info" size="16" />
               </div>
             </Text>
           </Tooltip>
-          <Input
-            style={inputStyle}
-            type="number"
-            placeholder="Your Initial Sale Price"
-            onChange={eventToValue(setNewPrice)}
```

Comment:
I would say it would be nice to still display this. But not have it as an editable value. But also happy to remove it for now!

Reply:
Discard the above comment, I wasnt looking properly. We should make a task to add units to all these values